ui/app/styles/core/switch.scss

```diff
@@ -12,7 +12,7 @@
   position: absolute;
   user-select: none;
 
-  +label {
+  + label {
     align-items: center;
     cursor: pointer;
     display: inline-flex;
```

Comment:
Nit: Maybe it's just the diff view but is the `label` class name on the same indentation level as the rules? It seems like it should be backed up.

Reply:
It's the diff. Here's how it looks in my editor. 
![image](https://user-images.githubusercontent.com/6618863/232634103-4c89112a-4b1d-4d3e-ba01-9aa50554f66b.png)